index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+</head>
+<body class="font-roboto">
+    <nav class="w-full gap-6 px-10 py-3 flex justify-end items-center">
+        <div class="cursor-pointer text-sm font-light hover:underline">Gmail</div>
+        <div class="cursor-pointer text-sm font-light hover:underline">Images</div>
+        <div class="cursor-pointer w-9 h-9 rounded-full hover:bg-gray-200 flex justify-center items-center">
+            <i class="bi bi-grid-3x3-gap-fill"></i>
+        </div>
+        <button class="bg-blue-600 text-white outline-none px-4 py-2 rounded-md hover:shadow-md font-medium">Sign in</button>
+    </nav>
+    <main class="flex flex-col items-center min-h-[80vh]">
+        <img src="./img/logo.png" alt="" class="w-52 lg:w-68">
+        <div class="border border-gray-300 rounded-3xl relative flex items-center px-4 mt-10 w-[90%] lg:w-2/5 hover:shadow-md">
+            <i class="bi bi-search text-gray-400 text-sm"></i>
+            <input type="text" class="outline-none bg-none border-none py-[10px] ml-3 w-full">
+            <img src="./img/mic.svg" alt="" class="w-5">
+        </div>
+        <div class="flex justify-center mt-6 gap-5 relative">
+            <button class="action-btn">Google Search</button>
+            <button class="action-btn">I'm Feeling Lucky</button>
+        </div>
+        <div class="flex justify-center-items-center gap-4 text-xs mt-6">
+            <div class="text-gray-700">Google offered in:</div>
+            <a href="#" class="lang-link">Français</a>
+            <a href="#" class="lang-link">Kiswahili</a>
+            <a href="#" class="lang-link">Kinyarwanda</a>
+        </div>
+    </main>
+
+    <footer class="bg-gray-100 w-full fixed bottom-0">
+        <div class="text-gray-500 px-10 py-3 text-sm">Your Country</div>
+        <div class="border w-full"></div>
+        <div class="footer--links flex justify-center flex-wrap gap-5 sm:justify-between px-10 py-3">
+            <div class="flex gap-8 flex-wrap justify-center sm:justify-start">
+                <div class="footer--link">About</div>
+                <div class="footer--link">Advertising</div>
+                <div class="footer--link">Business</div>
+                <div class="footer--link">How Search works</div>
+            </div>
+            <div class="flex gap-8 flex-wrap justify-center sm:justify-start">
+                <div class="footer--link">Privacy</div>
+                <div class="footer--link">Terms</div>
+                <div class="footer--link">Settings</div>
```

Comment:
according to the instructions use tailwindcss in your index.html remove those classes